test/transactions/4_registerMultisignatureAccount.js

```diff
@@ -348,10 +346,9 @@ describe('#registerMultisignatureAccount transaction', () => {
 			});
 
 			it('should have the asset with the multisignature with the min, lifetime and keysgroup', () => {
-				return registerMultisignatureTransaction.should.have
-					.property('asset')
-					.with.property('multisignature')
-					.with.properties('min', 'lifetime', 'keysgroup');
+				return registerMultisignatureTransaction.should.have.nested
+					.property('asset.multisignature')
+					.with.keys('min', 'lifetime', 'keysgroup');
```

Comment:
From the docs:
> Note that .all is used by default when neither .all nor .any appear earlier in the chain. However, it’s often best to add .all anyway because it improves readability.